extensions/resource-deployment/src/ui/notebookWizard/notebookWizardPage.ts

```diff
@@ -96,7 +96,12 @@ export class NotebookWizardPage extends ResourceTypePage {
 			this.wizard.wizardObject.generateScriptButton.hidden = true;
 		}
 
+		// by default the generate button is disabled and will wait for the page validation to be enabled.
+		this.wizard.wizardObject.generateScriptButton.enabled = false;
+
 		if (this.pageInfo.isSummaryPage) {
+			//generate button is always enabled for summary page as there are no in page validations.
+			this.wizard.wizardObject.generateScriptButton.enabled = true;
```

Comment:
But the summary page shouldn't have any validation errors because of this and so why is this needed to be special? This seems to indicate to me that something isn't being done correctly here.